configuration/pih/liquibase/liquibase.xml

```diff
@@ -144,7 +144,36 @@
             drop table if exists temp_value_drug;
         </sql>
     </changeSet>
-
+<changeSet id="20230201-update-programs-for-dead-patients-aali" author="aali">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+               select CASE WHEN count(*) > 0 THEN 1 ELSE 0 END AS cnt
+               from patient_program pp
+               inner join program p2 on p2.program_id = pp.program_id
+               where voided = 0
+               and date_completed  is null
+               and EXISTS ( select 1 from person p where p.person_id = pp.patient_id and p.dead = 1); 
+            </sqlCheck>
+        </preConditions>
+        <comment>If the count is 0, then remove these programs (Hypertension program, Diabetes program, Epilepsy program)</comment>
```

Comment:
Not sure exactly what the above comment means?

Reply:
Comment Updated, please check.